src/components/Nav/components/NavLinks/index.jsx

```diff
@@ -100,6 +101,16 @@ const GordonNavLinks = ({ onLinkClick }) => {
     />
   );
 
+  const LinksButton = (
+    <GordonNavButton
+      onLinkClick={onLinkClick}
+      linkName={'Links'}
+      linkPath={'/links'}
```

Comment:
When the value of a prop is just a string, you can pass it as a plain string:
```suggestion
      linkName="Links"
      linkPath="/links"
```